README.md

```diff
@@ -18,3 +18,185 @@ Requires:
+
+```
+export PYTHONPATH=SOME_PATH/PyGHee
+```
+
+## Step 5: Configuring and running EESSI bot on Linux machine
+
+You need to set up three environment variables: `GITHUB_TOKEN`, `GITHUB_APP_SECRET_TOKEN` and `GITHUB_PRIVATE_KEY`.
+
+### Step 5.1: GitHub Personal Access Token (PAT)
+
+Create a Personal Access Token (PAT) for your GitHub account via the page https://github.com/settings/tokens where you find a button "Generate new token".
+Give it meaningful name (field titled "Note") and set the expiration date. Then select the scopes this PAT will be used for. Then click "Generate token". On the result page, take note/copy the resulting token string -- it will only be shown once.
+
+On the Linux machine set the environment variable `GITHUB_TOKEN`, e.g.
+```
+export GITHUB_TOKEN='THE_TOKEN_STRING'
+```
+
+### Step 5.2: GitHub App Secret Token
+The GitHub App Secret Token is used to verify the webhook sender. You should have created one already when registering a new GitHub App in Step 1.
+
+On the Linux machine set the environment variable `GITHUB_APP_SECTRET_TOKEN`, e.g.
+```
+export GITHUB_APP_SECRET_TOKEN='THE_SECRET_TOKEN_STRING'
+```
+Note, depending on the characters used in the string you will likely have to use single quotes when setting the value of the environment variable.
+
+### Step 5.3: Create a private key and store it on the Linux machine
+The private key is needed to let the app authenticate when updating information at the repository such as commenting on PRs, adding labels, etc. You can create the key at the page of the GitHub App you have registered in Step 1.
+
+Open the page https://github.com/settings/apps and then click on the icon left to the name of the GitHub App for the EESSI bot or the "Edit" button for the app. Near the end of the page you will find a section "Private keys" where you can create a private key by clicking on the button "Generate a private key". The private key should be automatically downloaded to your local computer. Copy it to the Linux machine and note the full path to it.
+
+### Step 5.4: Obtain bot repository
+
+The bot needs a few scripts. These and an example configuration file are provided by the repository [EESSI/eessi-bot-software-layer](https://github.com/EESSI/eessi-bot-software-layer) (or your fork of it).
+
+First, clone the EESSI/eessi-bot-software-layer repository (or your fork of it) by running
+
+```
+git clone https://github.com/EESSI/eessi-bot-software-layer.git
+```
+
+### Step 5.5: Create the configuration file `app.cfg`
+
+After cloning the bot's repository, change directory with `cd eessi-bot-software-layer` and note the full path of the directory (`pwd`).
+
+If there is no `app.cfg` in the directory, create an initial version from `app.cfg.example`.
+
+```
+cp -i app.cfg.example app.cfg
+```
+
+Now set some values as follows:
```

Comment:
Should we also add some instructions about obtaining/setting `app_id` and `installation_id` here?

Reply:
Good catch. I had this on my TODO list.

Reply:
Added a section about this in commit https://github.com/EESSI/eessi-bot-software-layer/pull/2/commits/2524ba7e4c86feb51d828f5e665fc937502454e5